spouse/utils.py

```diff
@@ -46,3 +46,25 @@ def load_data() -> Tuple[
 
 def get_n_epochs() -> int:
     return 3 if IS_TEST else 30
+
+
+DATA_URL="https://www.dropbox.com/s/jmrvyaqew4zp9cy/spouse_data.zip?dl=1"
+FILES=( "train_data.pkl" "dev_data.pkl" "test_data.pkl" "dbpedia.pkl" )
+
+def download_files():
+    # Check that we are running from the right directory.
+    if os.path.split(os.getcwd())[1] != 'spouse':
+        raise Exception("Script must be run from spouse directory")
+    reload = False
+    for filename in FILES:
+        if not os.path.exists(filename):
+            reload = True
+    if reload:
+        if os.path.exists("data/"):
+            shutil.rmtree("data/")
+            os.mkdir("data")
+        urllib.request.urlretrieve(DATA_URL, "data.zip")       
+        with zipfile.ZipFile("data.zip", "r") as zip_ref:
+            zip_ref.extractall("data")
+        os.remove("data.zip")
```

Comment:
can we refactor this to a separate file that can be run from the CLI? 
that way, folks don't need to be _in_ the notebook in order to download the data

Reply:
also, any chance this script can be shared across different tutorials?